src/go/rpk/pkg/cli/topic/consume.go

```diff
@@ -763,6 +812,48 @@ func (c *consumer) intoOptions(topics []string) ([]kgo.Opt, error) {
 	return opts, nil
 }
 
+func (c *consumer) formatSchemaRegistryFlag() error {
+	if len(c.useSchemaRegistry) == 0 {
+		// We don't expect to hit this, but better be safe.
+		return errors.New("schema registry flag is empty")
+	}
+	for _, f := range c.useSchemaRegistry {
+		switch f {
+		case "val", "value":
```

Comment:
`case strings.HasPrefix("value", f)` ? Then I can use something like `--use-schema-registry=v` for extreme terseness

Reply:
Done.